pygfx/materials/_points.py

```diff
@@ -216,6 +216,123 @@ class PointsGaussianBlobMaterial(PointsMaterial):
     """
 
 
+class PointsMarkerMaterial(PointsMaterial):
```

Comment:
I presume that one of the challenges with transparent colors and edges might be that opacity and is_transparent are still independent of the colors and the antialiasing. 

I believe that with anti-aliasing enabled, is_transparent should always be set true and similarly for transparent colors. 

This is to ensure that it is rendered in the correct pass (opaque vs transparent).

Reply:
Oh good point! This crossed my mind a few times, but I forgot to update the render_mask logic!

Reply:
I think it might also be import for AA text but I haven't really looked into it

Reply:
I just checked. Could indeed take more into account:
https://github.com/pygfx/pygfx/blob/637bef6460dd47d7782c8e728ace69173c50e864/pygfx/renderers/wgpu/shaders/textshader.py#L73-L78

Reply:
pr on the way

Reply:
I see the `shaders/*.py` can add other properties that they look at when deciding what the `render_mask` is.